internal/table/table.go

```diff
@@ -246,9 +186,12 @@ func renderRow(providedColumns []string, colWidths []int) string {
 			suffix := strings.Repeat(" ", maxLen)
 
 			if len(columnEntry) > i {
-				repeat := maxLen - columnEntry[i].length
+				repeat := maxLen - columnEntry[i].Length
+				if repeat < 0 {
+					repeat = padding
+				}
```

Comment:
To simplify you could use http://github.com/ActiveState/cli/blob/master/internal/mathutils/mathutils.go#L9-L9

Reply:
Do you mean to use the max of `maxLen - columnEntry[i].Length` and `padding`? If so there are cases where the value we want will be the smaller of those two.

Reply:
I mean 

```
repeat := mathutils.MaxInt(0, maxLen - columnEntry[i].Length)
```

Reply:
Ahh okay. Thanks. For some reason I wasn't seeing this.